src/styles/pagination.m.css

```diff
@@ -0,0 +1,34 @@
+@import './shared/variables.css';
+
+.navigation {
```

Comment:
Sizing was addressed in the Grid PR so the body now "fills" the grid and will push the footer to the bottom